iphone/Maps/Core/Theme/Renderers/UISearchBarRenderer.swift

```diff
@@ -24,56 +24,54 @@ extension UISearchBar {
 class UISearchBarRenderer: UIViewRenderer {
   class func render(_ control: UISearchBar, style: Style) {
     super.render(control, style: style)
-    var searchTextField: UITextField?
     if #available(iOS 13, *) {
-      searchTextField = control.searchTextField
-    }
-    
-    // Default search bar implementation adds the grey transparent image for background. This code removes it and updates the corner radius. This is not working on iPad designed for mac.
-    if #available(iOS 14.0, *), ProcessInfo.processInfo.isiOSAppOnMac {
+      var searchTextField = control.searchTextField
+      // Default search bar implementation adds the grey transparent image for background. This code removes it and updates the corner radius. This is not working on iPad designed for mac.
+      if #available(iOS 14.0, *), ProcessInfo.processInfo.isiOSAppOnMac {
+      } else {
+        control.setSearchFieldBackgroundImage(UIImage(), for: .normal)
+      }
+      searchTextField.layer.setCorner(radius: 8)
+      searchTextField.layer.masksToBounds = true
+      // Placeholder color
+      if let placeholder = searchTextField.placeholder {
+        searchTextField.attributedPlaceholder = NSAttributedString(string: placeholder, attributes: [.foregroundColor: UIColor.gray])
+      }
+      if let backgroundColor = style.backgroundColor {
+        searchTextField.backgroundColor = backgroundColor
+      }
```

Comment:
Does this code have the same effect here and below?
```suggestion
      searchTextField.backgroundColor = style.backgroundColor
```

Reply:
`style.backgroundColor` can be nil. 
And passing nil into the opptional `backgroundColor` / `tintColor` will set it to the `clear`.
So optional unwrapping is need here.

Docs:
> Changes to this property can be animated. The default value is nil, which results in a transparent background color.

Reply:
Are there cases when these searchTextField properties should be set to nil to reset the color to a clear one?

Reply:
No. Because we are always use different than system's colours.